src/Plugin/RedirectPlugin.php

```diff
@@ -106,6 +106,7 @@ final class RedirectPlugin implements Plugin
      *
      *     @var bool|string[] $preserve_header True keeps all headers, false remove all of them, an array is interpreted as a list of header names to keep
      *     @var bool $use_default_for_multiple Whether the location header must be directly used for a multiple redirection status code (300).
```

Comment:
```suggestion
     *     @var bool $use_default_for_multiple Whether the location header must be directly used for a multiple redirection status code (300)
```

pretty-ci has weird rules on this construct, but lets keep it green